ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift

```diff
@@ -0,0 +1,148 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveUI
+import Lottie
+import Preferences
+import SwiftUI
+
+struct FocusSystemSettingsView: View {
+  @Environment(\.colorScheme) private var colorScheme
+  @Environment(\.verticalSizeClass) var verticalSizeClass: UserInterfaceSizeClass?
+  @Environment(\.horizontalSizeClass) var horizontalSizeClass: UserInterfaceSizeClass?
+
+  @Binding var shouldDismiss: Bool
+
+  var shouldUseExtendedDesign: Bool {
+    return horizontalSizeClass == .regular && verticalSizeClass == .regular
+  }
+
+  let dynamicTypeRange = (...DynamicTypeSize.xLarge)
+
+  var body: some View {
+    NavigationView {
+      if shouldUseExtendedDesign {
+        VStack(spacing: 40) {
+          VStack {
+            settingsSystemContentView
+              .background(colorScheme == .dark ? .black : .white)
+          }
+          .clipShape(RoundedRectangle(cornerRadius: 16.0))
+          .frame(maxWidth: 616, maxHeight: 895)
+          .shadow(color: .black.opacity(0.1), radius: 18, x: 0, y: 8)
+          .shadow(color: .black.opacity(0.05), radius: 0, x: 0, y: 1)
+
+          FocusStepsPagingIndicator(totalPages: 4, activeIndex: .constant(3))
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .background(Color(braveSystemName: .pageBackground))
+      } else {
+        VStack(spacing: 16) {
+          settingsSystemContentView
+          FocusStepsPagingIndicator(totalPages: 4, activeIndex: .constant(3))
+        }
+        .padding(.bottom, 20)
+        .background(Color(braveSystemName: .pageBackground))
+      }
+    }
+    .navigationViewStyle(StackNavigationViewStyle())
+    .navigationBarHidden(true)
+  }
+
+  private var settingsSystemContentView: some View {
+    VStack {
+      VStack {
+        VStack(spacing: 10) {
+          Text(Strings.FocusOnboarding.defaultBrowserScreenTitle)
+            .font(
+              Font.custom("FlechaM-Medium", size: 32)
+            )
+            .dynamicTypeSize(dynamicTypeRange)
+            .fixedSize(horizontal: false, vertical: true)
+            .multilineTextAlignment(.center)
```

Comment:
move repeated modifiers to parent

Reply:
All duplicates moved to root https://github.com/brave/brave-core/commit/91c6c912a8207d7f813f771f7e0e16ff5050eb1d